File: youlexuan-sellergoods-interface/src/main/java/com/offcn/sellergoods/service/BrandService.java
package com.offcn.sellergoods.service;

import com.offcn.entity.PageResult;
import com.offcn.pojo.TbBrand;

import java.util.List;
import java.util.Map;

public interface BrandService {
    //查询所有类
    public List<TbBrand> getAll();

    //添加Brand
    public void add(TbBrand tbBrand);

    //根据Id查找对应的实体类
    public TbBrand findOne(Long id);

    //跟据id修改实体类
    public void update(TbBrand tbBrand);

    //删除实体类
    public void deleteId(Long[] ids);

    //分页查询
    public PageResult findpage(TbBrand tbBrand,int pageNume,int pageSize);

    //返回json串
    public List<Map> selectOptionList();

}

File: youlexuan-manager-web/src/main/webapp/js/service/brandService.js
app.service("brandService",function ($http) {
			this.fandAll=function () {
				return 	$http.get('../brand/findAll.do');
			};
			this.add=function (entity) {
				return $http.post('../brand/add.do',entity);
			};
			this.update=function (entity) {
				return $http.post('../brand/update.do',entity);
			};
			this.search=function (page,rows,searchEntity) {
				return $http.post("../brand/search.do?page="+page+"&rows="+rows,searchEntity);
			};
			this.deleteId=function (selectIds) {
				return $http.get('../brand/deleteId.do?ids='+selectIds);
			}
			this.selectOptionList=function () {
            	return $http.get('../brand/selectOptionList.do');
            }

});